build/build-component-lists.php

```diff
@@ -122,10 +122,17 @@ function injectProjectContent(string $content, string $file) : void
 {
     $homepage = file_get_contents($file);
     $replacement = preg_replace(
-        '#(?<start>\<\!-- START COMPONENT LISTS --\>).*?(?<end>\<\!-- END COMPONENT LISTS --\>)#s',
-        '$1' . $content . '$2',
+        [
+            '#(?<start>\<\!-- START COMPONENT LISTS --\>).*?(?<end>\<\!-- END COMPONENT LISTS --\>)#s',
+            '#(?<start>\<\!-- START YEAR --\>).*?(?<end>\<\!-- END YEAR --\>)#s',
+        ],
+        [
+            '$1' . $content . '$2',
+            '${1}' . date('Y') . '${2}',
```

Comment:
But have I overlooked something, or why are brackets used in some places and not in others?